src/aleph/vm/conf.py

```diff
@@ -127,6 +127,11 @@ class Settings(BaseSettings):
     WATCH_FOR_UPDATES = True
 
     API_SERVER = "https://official.aleph.cloud"
+    # Connect to the Quad9 VPN provider using their IPv4 and IPv6 addresses.
+    CONNECTIVITY_IPV4_URL = "https://9.9.9.9/"
+    CONNECTIVITY_IPV6_URL = "https://[2620:fe::fe]/"
+    CONNECTIVITY_DNS_HOSTNAME = "example.org"
```

Comment:
https://www.iana.org/help/example-domains
Maybe adding fallback addresses to test would be useful?
> We provide a web service on the example domain hosts to provide basic information on the purpose of the domain. These web services are provided as best effort, but are not designed to support production applications. While incidental traffic for incorrectly configured applications is expected, please do not design applications that require the example domains to have operating HTTP service.